llpc/test/shaderdb/general/PipelineRays_TestLgcRtTraceRayOp.pipe

```diff
@@ -0,0 +1,98 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc --print-after=llpc-spirv-lower-translator -gfxip 10.3 -o /dev/null 2>&1 %s | FileCheck -check-prefixes=SHADERTEST %s
+; SHADERTEST-LABEL: @main(
+; SHADERTEST: call void (...) @lgc.rt.trace.ray
```

Comment:
Can you at least verify the arguments are being properly added to the trace ray operation?

Reply:
Sure. 

However the PR check says the test failed but I cannot reproduce on my local so far, still investigating.